parts/language/lang_ja.lua

```diff
@@ -871,14 +871,14 @@ getTip={refuseCopy=true,
         "1next 1hold!",
         "1next 6hold!",
         "20Gは全く新しい種類のゲームルールです!",
-        "20PCできるかい?",
-        "26TSDできるかい?",
-        "2つの回転を使ってみよう、3つ使うとさらにいいです",
+        "20PCって何?",
+        "26TSDって何?",
+        "2つの回転を使ってみよう、3つ使うとさらにいいです!",
```

Comment:
This reverts some changes introduced in #589 . Is this intentional or accidental?
**If** this is by accident, there can be possibly other places that are changed unintentionally. Maybe...we could check again and add another new pull request to fix accidental changes? Do you think it is necessary?

Reply:
Oh whoops, line 874 and 875 should not be like that. :P
Sorry for unintentional changes...